app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,18 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+    def __init__(self, nickname: str,
+                 hummer_level: int,

```

Comment:
Code Style: The variable name 'hummer_level' is a bit confusing. It might be better to use a more descriptive name, such as 'hammer_skill_level' or 'hammer_power_level'.